GraphQL/resolvers.py

```diff
@@ -0,0 +1,62 @@
+import httpx
+
+async def get_all_subscriptions_resolver():
+    from schema import Subscription
+    async with httpx.AsyncClient() as client:
+        response = await client.get("http://fastapi_service:8000/subscriptions")
+        raw_subscriptions = response.json().get("subscriptions", [])
```

Comment:
You need to have a check to determine if the request passed successfully and return the respective error if it was not.